mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/util/generation/GeneratedCodePatterns.java

```diff
@@ -333,12 +337,17 @@ public static void setValue(
               targetName,
               cqlName,
               rawCollectionName,
-              enclosingClass.addGenericTypeConstant(type.asRawTypeName()))
-          .nextControlFlow(
-              "else if ($L == $T.$L)",
-              NULL_SAVING_STRATEGY,
-              NullSavingStrategy.class,
-              NullSavingStrategy.SET_TO_NULL)
+              enclosingClass.addGenericTypeConstant(type.asRawTypeName()));
+      if (useNullSavingStrategy) {
+        methodBuilder.nextControlFlow(
+            "else if ($L == $T.$L)",
+            NULL_SAVING_STRATEGY,
+            NullSavingStrategy.class,
+            NullSavingStrategy.SET_TO_NULL);
+      } else {
+        methodBuilder.nextControlFlow("else");
+      }
```

Comment:
nit: This code is the same as:
https://github.com/datastax/java-driver/pull/1415/files#diff-8992e8a0bae64eda9aaf4ff5012fa95fR300-R309
maybe we could extract method?

Reply:
It's just 3 lines of code, it doesn't buy much and I think it would make the code harder to read.